doc/man3/EVP_MAC.pod

```diff
@@ -344,6 +338,10 @@ look like this:
 (in this example, that program was stored in F<foo.c> and compiled to
 F<./foo>)
 
+=head1 HISTORY
+
+The functions are added in OpenSSL 3.0.0.
```

Comment:
`These functions were added in `...

At the time of release, past tense will make sense.